packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/table-of-contents-hierarchy~no-colored.md

```diff
@@ -0,0 +1,49 @@
+### Description
```

Comment:
Can you consolidate this doc into mayflower/packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/table-of-contents-hierarchy.md

Reply:
I removed that one since essentially was the same file.